scripts/build/termux_step_install_license.sh

```diff
@@ -53,36 +53,41 @@ termux_step_install_license() {
 				|'curl'|'HPND'|'ISC'|'Libpng'|'MIT'|'Openfont-1.1'\
 				|'PythonPL'|'X11'|'ZLIB')
 				# Find the license file in the source files
+				local FOUND=0
 				for FILE in "${COMMON_LICENSE_FILES[@]}"; do
 					[[ -f "$TERMUX_PKG_SRCDIR/$FILE" ]] && {
 						if (( COUNTER )); then
 							cp -f "${TERMUX_PKG_SRCDIR}/$FILE" "${TERMUX_PREFIX}/share/doc/${TERMUX_PKG_NAME}/copyright.${COUNTER}"
 						else
 							cp -f "${TERMUX_PKG_SRCDIR}/$FILE" "${TERMUX_PREFIX}/share/doc/${TERMUX_PKG_NAME}/copyright"
 						fi
-						# since this is a post-increment, (( 0 )) would be falsey
-						# thus `set -e` would kill the script on the first iteration
-						# using `true` prevents this
-						: $(( COUNTER++ ))
+						(( ++FOUND ))
 					}
+					if (( ! FOUND )); then
+						# If we get here no license file could be found
+						termux_error_exit "${TERMUX_PKG_NAME}: Could not find a license file for $LICENSE in the package sources"
+					fi
```

Comment:
This is gonna fail on the first license file not found. It needs to be after `for`.

I wonder why we don't just break out of the for loop if a common license file is found and instead look for all possible variants.

https://github.com/termux/termux-packages/commit/dcb926abd49ea66e02b642de722d4997e22a326f#diff-a5603dfea6540f0f85a612bca35a280e3fec07bb6e66171f0d67fad2d2b523e8

Reply:
> This is gonna fail on the first license file not found. It needs to be after `for`.

Is it not?

Reply:
Since you have now moved the counter increment at the end of the while loop instead of keeping it inside respective `if` when a license is installed like I asked before, its not gonna get incremented for multiple common license files installed and later ones will overwrite older ones.

Reply:
Yep I get it now.
Alright, going with your plan.